package.json

```diff
@@ -55,7 +55,8 @@
     "uuid": "^10.0.0",
     "viem": "^2.18.1",
     "xstate": "^5.18.2",
-    "zustand": "^5.0.0-rc.2"
+    "zustand": "^5.0.0-rc.2",
+    "@lifeomic/attempt" : "3.1.0"
```

Comment:
Please reinstall using `yarn add @lifeomic/attempt`, it will write correct version and put in right order